content/2023/contents/home.mdx

```diff
@@ -51,3 +55,33 @@ together for good and open computational sciences.
     <img src="https://live.staticflickr.com/65535/52919230990_ec629d4edb_k.jpg" />
   </div>
 </div>
+
+# Sponsorship
+
+<div className="tw-flex tw-flex-col tw-items-center tablet:tw-flex-row tablet:tw-justify-center tablet:tw-items-start tablet:tw-h-60">
```

Comment:
the image is too close to the Partnership header text, could you please adding some padding here?

Reply:
added c1b1e913ab02d142a354e67ef5531b9d1ba1c314